src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,74 @@
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is no UI configuration at this time, and APM auto-injected agents are not able to use this until further improvements are made.
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch chunks or send data.
+
+For example, using copy and paste:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-and-paste installation to fetch agent chunks. This implies you are hosting the file somewhere instead it being retrieved via our CDN. 
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.___-1.240.min.js` file.
+* If you are hosting the file on a `nrassets.abc.com` proxy, you must provide this value as the URL, `'https://nrassets.abc.com/'`, including the scheme and ending slash.
+
+This is not relevant to NPM installed agents, as the main host project's bundler should take care of resources. `init.proxy.beacon` is used by both installation to determine the host to which data such as page view timings are sent. 
+
+* For example, if those payloads are routed through a `nrbeacon.abc.com` proxy, then that is the exact string value to use. 
+* Note that the scheme (e.g, `'https://'`) and ending slash **should be omitted** for this property.
+
+```js
+NREUM.init = {
+    proxy: {
+        assets: 'https://exampleproxy.com/',
+        beacon: 'exampleproxy.com'
+    }
+}
+```
+
+## About your proxy server [#expectations]
+
+The New Relic browser agent team provides neither a cloud proxy service nor support on such service application. **You must make and maintain your own proxy servers.**
+
+To that effect, some assumptions are made about your proxy that must hold true for full functionality:
+
+1. As mentioned above, if a proxy is used to load the agent chunk(s), it must hold an exact replica of that(those) chunk(s). E.g., for v1.240.0, `
+https://js-agent.newrelic.com/nr-spa.3b61d78f-1.240.0.min.js` should duplicatively exist at `
+https://yourproxy.com/nr-spa.3b61d78f-1.240.0.min.js`.
+2. No modification to the original client outgoing payload is made by the proxy.
+3. The proxy forwards client payloads to New Relic, e.g. `https://bam.nr-data.net`.
+4. Responses by the New Relic endpoints are also forwarded back to client unmodified.
```

Comment:
You could simplify 2 and 6.

2. Requests and responses through the proxy server should not be modified.